components/dialogs/AuthDialog.vue

```diff
@@ -325,6 +342,18 @@ export default {
     },
   },
   async mounted() {
+    if (this.shouldShowFromUserIcon) {
```

Comment:
Is `shouldShowFromUserIcon` ready when executing `mounted()`?

Reply:
`shouldShowFromUserIcon` is populated by nuxt plugin, thus should be always ready